ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/ospf-cleartext-passwords.yml

```diff
@@ -0,0 +1,275 @@
+---
+# Tests using cleartext OSPF passwords - verify they are automatically encrypted
+
+type: l3leaf
+
+l3leaf:
+  defaults:
+    loopback_ipv4_pool: 192.168.66.0/24
+    vtep_loopback_ipv4_pool: 192.168.142.0/24
+    virtual_router_mac_address: 00:11:22:33:44:55
+    bgp_as: 666
+  nodes:
+    - name: ospf-cleartext-passwords
+      id: 42
+      mgmt_ip: 192.168.42.42/24
+
+tenants:
+  - name: Tenant_OSPF
+    mac_vrf_vni_base: 10000
+    mac_vrf_id_base: 20000
+    redistribute_mlag_ibgp_peering_vrfs: true
+    vrfs:
+      - name: Tenant_OSPF_VRF_LEVEL_SIMPLE_ENCRYPTION
+        vrf_vni: 100
+        vrf_id: 42
```

Comment:
This test is using vrf_id as OSPF process ID, I'd consider additional test without vrf_id and OSPF process_id defined explicitly. This is low priority however, as IDs are probably tested somewhere else.

Reply:
yeah thats' taken care of somewhere else. won't fix :)